contrib/ansible/inventory/hosts.localhost

```diff
@@ -4,6 +4,7 @@ agents
 
 [skydive:vars]
 skydive_listen_ip=0.0.0.0
+skydive_etcd_port=12345
```

Comment:
12379 ?

Reply:
that an example, We can put anything. Anyway I will remove this line as it was to do my own tests